packages/address/src/AddressStyles.tsx

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components'
+
+export const AddressStyle = styled.div`
+  position: relative;
+  display: inline-block;
+  font-weight: 500;
+  margin: 0 6px;
```

Comment:
These margins was useful for the IdenticonBadge address for some address indentation inside the wrapper:

![image](https://user-images.githubusercontent.com/1422379/128463458-b2bfbe05-c715-47a1-8d06-984b4544e7ee.png)

If the Address component is in other places, it will be unnecessary indents. I think it's better to bring them back to the IdenticonBadge component.